pxtlib/hf2.ts

```
@@ -270,6 +280,13 @@ namespace pxt.HF2 {
             this.eventHandlers[id + ""] = f
         }
 
+        sendCustomEventAsync(type: string, payload: Uint8Array): Promise<void> {
+            if (type == "jacdac")
+                return this.talkAsync(HF2_CMD_JDS_SEND, payload)
+                    .then(() => { })
```

what happens when we fail, just swallow the error?